tests/integration/test_project_navigator.py

```diff
@@ -288,7 +288,7 @@ def submissions(self):
 
     @property
     def image_path(self):  # pylint: disable=no-self-use
-        return os.path.join(os.path.split(__file__)[0], 'example-image.gif')
+        return os.path.join(os.path.split(__file__)[0], "../resources/", 'sample_upload.png')
```

Comment:
You probably already saw this, but tests are breaking because this image has been removed in the latest commit